test/uk/gov/hmrc/economiccrimelevyreturns/navigation/UkRevenuePageNavigatorSpec.scala

```diff
@@ -33,16 +36,51 @@ class UkRevenuePageNavigatorSpec extends SpecBase {
   val pageNavigator = new UkRevenuePageNavigator(mockEclLiabilityService)
 
   "nextPage" should {
-    "return a Call to the Aml regulated activity for full financial year page in NormalMode" in forAll {
-      (eclReturn: EclReturn, ukRevenue: Long) =>
-        val updatedReturn = eclReturn.copy(relevantApRevenue = Some(ukRevenue))
+    "return a Call to the Aml regulated activity for full financial year page in NormalMode when the calculated band size is not Small" in forAll(
+      arbEclReturn.arbitrary,
+      Gen.chooseNum[Long](UkRevenueThreshold, MinMaxValues.RevenueMax),
```

Comment:
I don't think the navigators actually care about the long that you generate as we mock the response from the calculation so you could probably just do `Arbitrary.arbitrary[Long]` instead of this